core/app/models/spree/address.rb

```diff
@@ -161,28 +160,5 @@ def country_iso=(iso)
     def country_iso
       country && country.iso
     end
-
-    # @return [String] the full name on this address
-    def name
-      Spree::Address::Name.new(
-        read_attribute(:firstname),
-        read_attribute(:lastname)
-      ).value
-    end
-
-    def name=(value)
-      return if value.nil?
-
-      name_from_value = Spree::Address::Name.new(value)
-      write_attribute(:firstname, name_from_value.first_name)
-      write_attribute(:lastname, name_from_value.last_name)
-    end
-
-    def as_json(options = {})
```

Comment:
@filippoliverani so address's JSON representation will still contain firstname and lastname?

Reply:
We added `Spree::Address.ignore_columns` to address this.